src/utils/api/client.ts

```diff
@@ -3,11 +3,16 @@ import { Millisecond } from "@pagopa/ts-commons/lib/units";
 import * as E from "fp-ts/Either";
 import { pipe } from "fp-ts/function";
 import { createCounter } from "../../utils/counter";
-import { createClient as createIOClient } from "../../../generated/definitions/payment-ecommerce-webview/client";
-import { createClient as createCHECKOUTClient } from "../../../generated/definitions/payment-ecommerce/client";
+import { createClient as createIOClient } from "../../../generated/definitions/payment-ecommerce-webview-v2/client";
```

Comment:
```suggestion
import { createClient as createIOClientV2 } from "../../../generated/definitions/payment-ecommerce-webview-v2/client";
```

just make explicit that it is v2 client version from the alias name itself

Reply:
done here 4b7439206a60442e886bd88991650f16ba7b4d5a